backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/materialDispo/forecast.feature

```diff
@@ -0,0 +1,92 @@
+    And the PP_Product_BOM identified by bom_1 is completed
+    And metasfresh contains PP_Product_Plannings
+      | Identifier | M_Product_ID.Identifier | OPT.PP_Product_BOMVersions_ID.Identifier | IsCreatePlan |
+      | ppln_1     | p_1                     | bomVersions_1                            | false        |
+
+    And metasfresh contains C_BPartners:
+      | Identifier    | Name                   | OPT.IsVendor | OPT.IsCustomer | M_PricingSystem_ID.Identifier |
+      | endcustomer_1 | EndCustomer_03032025_6 | N            | Y              | ps_1                          |
+
+    And load M_Warehouse:
+      | M_Warehouse_ID.Identifier | Value        |
+      | warehouseStd              | StdWarehouse |
+    And metasfresh contains M_HU_PI:
+      | Identifier         |
+      | huPackingLU        |
+      | huPackingTU        |
+      | huPackingVirtualPI |
+    And metasfresh contains M_HU_PI_Version:
+      | M_HU_PI_Version_ID.Identifier | M_HU_PI_ID.Identifier | Name             | HU_UnitType | IsCurrent |
+      | packingVersionLU              | huPackingLU           | packingVersionLU | LU          | Y         |
+      | packingVersionTU              | huPackingTU           | packingVersionTU | TU          | Y         |
+      | packingVersionCU              | huPackingVirtualPI    | No Packing Item  | V           | Y         |
+    And metasfresh contains M_HU_PI_Item:
+      | M_HU_PI_Item_ID.Identifier | M_HU_PI_Version_ID.Identifier | Qty | ItemType | OPT.Included_HU_PI_ID.Identifier |
+      | huPiItemLU                 | packingVersionLU              | 10  | HU       | huPackingTU                      |
+      | huPiItemTU                 | packingVersionTU              | 0   | MI       |                                  |
+    And metasfresh contains M_HU_PI_Item_Product:
+      | M_HU_PI_Item_Product_ID.Identifier | M_HU_PI_Item_ID.Identifier | M_Product_ID.Identifier | Qty | ValidFrom  |
+      | huItemPurchaseProduct              | huPiItemTU                 | p_1                     | 10  | 2020-02-01 |
+    And metasfresh contains M_Forecasts:
+      | Identifier | Name            | DatePromised | M_Warehouse_ID.Identifier |
+      | f_1        | test_03032025_6 | 2021-04-17   | warehouseStd              |
+    And metasfresh contains M_ForecastLines:
+      | Identifier | M_Forecast_ID.Identifier | M_Product_ID.Identifier | Qty | M_Warehouse_ID.Identifier | C_UOM_ID.X12DE355 |
+      | fl_1       | f_1                      | p_1                     | 10  | warehouseStd              | PCE               |
+    When the forecast identified by f_1 is completed
+    Then after not more than 60s, the MD_Candidate table has only the following records
+      | Identifier | MD_Candidate_Type | OPT.MD_Candidate_BusinessCase | M_Product_ID.Identifier | DateProjected        | Qty | Qty_AvailableToPromise | OPT.DateProjected_LocalTimeZone |
```

Comment:
remove "OPT." and ".Identifier".

NOTE: use distribution.feature as an example